wiki_doc

```diff
@@ -0,0 +1,190 @@
+## Valkey Commands
+
+For information on the supported commands and their corresponding parameters, we recommend referring to the documentation in the code. This documentation provides in-depth insights into the usage and options available for each command.
+
+### Command String Arguments
+
+Valkey stores strings as sequences of bytes, so that may include text, serialized objects, or binary arrays. As such, to pass Valkey strings as arguments to commands, or receive Valkey strings in responses, GLIDE offers `GlideString`, which represents two APIs:
+
+1. `string`: an UTF-8 string.
+2. `Buffer`: an array of bytes.
+
+```typescript
+/**
+ * Union type that can store either a valid UTF-8 string or array of bytes.
+ */
+export type GlideString = string | Buffer;
+```
+
+The return types of command API:
+
+-   Command uses `string` as the return type if it constains a specific string value, which could never be binary data. For example, `type` command returns one of pre-defined strings, or `set` command which returns `OK`.
```

Comment:
```suggestion
-   Command uses `string` instead of GlideString as the return type if it contains a simple string value (simple strings in Valkey are always UTF-8 encoded strings). For example, `type` command returns one of pre-defined strings, or `set` command which returns `"OK"`.
```

Reply:
Add links to the commands for `type` and `set`.